File: 1045.py
x = input().split(' ')
lista = [float(x[0])];lista+=[float(x[1])];lista+=[float(x[2])]
lista.sort(); a=lista[2];b = lista[1]; c=lista[0]
sinal = 0
if a >= b+c:
    print('NAO FORMA TRIANGULO')
    sinal = 1
if a**2 == b**2 + c**2 and sinal==0:
    print('TRIANGULO RETANGULO')
if a**2 > b**2 + c**2 and sinal==0:
    print('TRIANGULO OBTUSANGULO')
if a**2 < b**2 + c**2 and sinal==0:
    print('TRIANGULO ACUTANGULO')
if a==b==c and sinal==0:
    print('TRIANGULO EQUILATERO')
if a==b!=c or a==c!=b or b==c!=a and sinal==0:
    print('TRIANGULO ISOSCELES')

File: 1045.java
import java.util.Scanner;
import java.util.Arrays;
import java.util.Locale;
public class Main {
public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);
        double [] lista = new double[3]; double a, b, c;
        lista[0] = input.nextDouble(); lista[1] = input.nextDouble(); lista[2] = input.nextDouble();
        Arrays.sort(lista); a = lista[2]; b = lista[1]; c = lista[0];
        int sinal = 0;
        double n = Math.pow(b,2) + Math.pow(c,2);
        if(a >= b+c) {
                System.out.println("NAO FORMA TRIANGULO");
                sinal = 1;
        }
        if(Math.pow(a,2) == n && sinal == 0) {
                System.out.println("TRIANGULO RETANGULO");
        }
        if(Math.pow(a,2) > n && sinal == 0) {
                System.out.println("TRIANGULO OBTUSANGULO");
        }
        if(Math.pow(a,2) < n && sinal == 0) {
                System.out.println("TRIANGULO ACUTANGULO");
        }
        if(a == b && b == c && sinal == 0) {
                System.out.println("TRIANGULO EQUILATERO");
        }
        if(a == b && b!= c || a == c && c !=b || b == c && c !=a && sinal == 0) {
                System.out.println("TRIANGULO ISOSCELES");
        }
}
}
